corehq/apps/accounting/forms.py

```diff
@@ -266,10 +281,28 @@ def clean_email_list(self):
     def clean_enterprise_admin_emails(self):
         # Do not return a list with an empty string
         if self.cleaned_data['enterprise_admin_emails']:
-            return self.cleaned_data['enterprise_admin_emails'].split(',')
+            return [e.strip() for e in self.cleaned_data['enterprise_admin_emails'].split(r',')]
         else:
             return []
 
+    def clean_enterprise_restricted_signup_domains(self):
+        if self.cleaned_data['enterprise_restricted_signup_domains']:
```

Comment:
Are there restrictions on which email domains an account can claim? This seems like a lot of power to have in a self-service dashboard, even for enterprise clients

Reply:
This input isn't on the client dashboard, it's on the account settings page, which is only visible to Dimagi internal billing admins (e.g., Julie). Because yes, totally agree clients shouldn't be able to block arbitrary domains.

Reply:
Ah great, I was missing that. I figured this was a dashboard for enterprise clients to have more control over their domains, but that makes sense. Thanks for explaining!